ci/functional/test_main_node.sh

```diff
@@ -11,4 +11,14 @@ export REMOTE_ACCT=jenkins
 export WITH_VALGRIND="$WITH_VALGRIND"
 export STAGE_NAME="$STAGE_NAME"
 
+if [ -n "$BULLSEYE" ]; then
+    pushd "${SL_SRC_DIR}/bullseye"
+    set +x
+    echo + sudo ./install --quiet --key "**********" --prefix /opt/BullseyeCoverage
+    sudo ./install --quiet --key "${BULLSEYE}" --prefix /opt/BullseyeCoverage
+    set -x
+    popd
+    rm -rf bullseye
+    export PATH="/opt/BullseyeCoverage/bin:$PATH"
```

Comment:
With the addition of the vars/configureBullseye.groovy file this should change to:
```suggestion
    export PATH="/opt/BullseyeCoverage/bin:$PATH"
```

and we chould change https://github.com/daos-stack/daos/blob/master/ci/unit/test_main_node.sh#L24-L35 to be:
```
if [ -n "$BULLSEYE" ]; then
    export COVFILE="${SL_SRC_DIR}/test.cov"
    export PATH="/opt/BullseyeCoverage/bin:$PATH"
    log_prefix="covc_test"
fi
```